cql-parser/src/test/scala/troy/cql/parser/dml/SelectStatementParserTest.scala

```diff
@@ -323,7 +323,7 @@ class SelectStatementParserTest extends FlatSpec with Matchers {
     relations.size shouldBe 1
     relations(0).asInstanceOf[Relation.Simple].columnName shouldBe "weight"
     relations(0).asInstanceOf[Relation.Simple].operator shouldBe Operator.Equals
-    relations(0).asInstanceOf[Relation.Simple].term.asInstanceOf[FloatConstant].value shouldBe "50.4"
+    relations(0).asInstanceOf[Relation.Simple].term.asInstanceOf[FloatConstant].value shouldBe 50.4
```

Comment:
I searched in the scalatest repo and there is many issues about Float numbers, I am not sure if this is a bug or something else.